src/containers/Tag/TagList/TagList.module.css

```diff
@@ -1,8 +1,87 @@
+.Buttons {
```

Comment:
I added a lot of styling, which wasn't necessarily needed for Pagination to work properly, but I'm always eager to use styling to make webpages look nice, so I figured I could add a little bit of it in my components here.

Reply:
Perfectly fine for now. The look and feel might change once we have design ready.